devicesetup/src/main/res/layout/activity_create_account.xml

```diff
@@ -38,6 +38,22 @@
                 tools:ignore="MissingPrefix"/>
 
             <EditText
+                android:id="@+id/first"
+                style="@style/SparkEditText"
+                android:layout_width="match_parent"
+                android:layout_marginBottom="@dimen/edittext_form_element_gap"
+                android:hint="@string/prompt_first_name"
+                android:inputType="text"/>
```

Comment:
These name fields `inputType` should be `textPersonName` or (better yet) `textPersonName|textCapWords`